Tasks/Common/webdeployment-common/fileTransformationsUtility.ts

```diff
@@ -45,6 +47,10 @@ export async function fileTransformations(isFolderBasedDeployment: boolean, JSON
         jsonSubstitutionUtility.jsonVariableSubstitution(folderPath, JSONFiles);
         console.log(tl.loc('JSONvariablesubstitutionappliedsuccessfully'));
     }
+}
+
+export async function archiveFolder(isFolderBasedDeployment: boolean, webDeployPkg: string, folderPath: string) {
```

Comment:
same as above comment. 
use ziputility.archiveFolder() and remove the logic from this file.